pilot-app/src/components/form/fields/wimTextArea.vue

```diff
@@ -0,0 +1,52 @@
+﻿<template>
+    <div :class="classname">
+        <textarea cols="32"
+                  rows="3"
+                  type="text"
+                  v-model="field.value"
+                  v-on="eventHandler(field, 'handleChange')"
+                  :class="fieldClass(field)"
+                  :name="field.propertyName"
+                  :id="field.propertyName"
+                  v-bind:disabled="field.disabled || field.readOnly"
+                  ref="textArea">{{field.value}}</textarea>
+    </div>
+</template>
+<script>
+    import { fieldMixin } from './../form';
+
+    export default {
+        props: ['field', 'classname'],
+        mixins: [fieldMixin],
+        data() {
+            return {
+                offset: undefined,
+            }
+        },
+        methods: {
+            handleChange($event) {
+                console.log('triggerChange:textarea');
+                this.$emit('onchange', $event, this.field);
+            },
+            autoResize(element) {
+                element.style.height = 'auto';
+                element.style.height = element.scrollHeight + this.offset + 'px';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8170341)
```suggestion
                element.style.height = element.scrollHeight + this.offset +"px"';
```